### PR TITLE
Updated to mount the /boot partition

### PR DESCRIPTION
**Pull request by mns6070**
Added steps to mount and unmount the /boot partition during the repair process.

### CODE REVIEW

**Comment by mvincerx** [comment]
@mns6070 Thanks for continuing to review our docs and contribute.
@bktan8. This seems like a logical approach. Could you please review this as you're the original author?

File: source/guides/maintenance/fix-broken-install.rst [hunk 1]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
 
 #. Ensure the system is connected to the network.
 
-#. Mount the system’s root partition.
+#. Mount the system’s root and boot partitions.
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition.  So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 2]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
...
 
-   #. To find the root partition, run:
+   #. To find the root and boot partitions, run:
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 3]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
...
 
       .. code-block:: bash
 
-         lsblk
+         lsblk -o NAME,LABEL,PARTTYPE,PARTLABEL
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: Good suggestion.

File: source/guides/maintenance/fix-broken-install.rst [hunk 4]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
...
 
-      We'll use :file:`/dev/sda3/` as the root partition example.
+      We'll use :file:`/dev/sda3/` as the root partition example and :file:`/dev/sda1/` as the boot partition example.
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 5]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
...
 
-   #. Next, mount the partition to the :file:`/mnt` folder.
+   #. Next, mount the root partition to the :file:`/mnt` folder and the boot partition to the :file:`/mnt/boot` folder.
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 6]
```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
...
 
       .. code-block:: bash
 
          sudo mount /dev/sda3 /mnt
+         sudo mount /dev/sda1 /mnt/boot
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 7]
```diff
@@ -75,10 +76,11 @@ Mount root partition, verify, and fix
 
    :ref:`Learn more about how swupd works <swupd-guide>`.
 
-#. After the process is complete, unmount the root partition:
+#. After the process is complete, unmount the boot and root partitions:
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst [hunk 8]
```diff
@@ -75,10 +76,11 @@ Mount root partition, verify, and fix
...
 
    .. code-block:: bash
 
+      sudo umount /mnt/boot
```
Comment (bktan8) [inline source/guides/maintenance/fix-broken-install.rst]: swupd does not manage the EFI partition. So we it's not necessary to mount and unmount the boot partition.

File: source/guides/maintenance/fix-broken-install.rst (hunk 3 above)
Comment (mvincerx) [reply source/guides/maintenance/fix-broken-install.rst]: I agree. This is helpful in viewing the output. In summary, please delete all other changes on a new commit to this PR, and then we'll accept for merge. Cheers, @mns6070 .

**Comment by mns6070** [comment]
What happens if there is a kernel update from a newer release level?  Doesn't swupd try to update the kernel by calling clr-boot-manager, which does manage the EFI boot partition?

**Comment by bktan8** [comment]
Correct.  clr-boot-manager manages the EFI partition.

**Comment by mns6070** [comment]
> Correct. clr-boot-manager manages the EFI partition.

Just want to make sure I understand the workflow completely.  If someone runs the swupd repair command referenced here without /boot EFI parition mounted and there is an update to the kernel, swupd updates the kernel and calls clr-boot-manager to finish the kernel update.  At that point if /boot is not mounted, clr-boot-manager writes the changes to /boot, which would be on the root partition and not on the EFI partition because Clear Linux is booted from the recovery media and not from the local disk, correct?

**Comment by bktan8** [comment]
"swupd repair" doesn't do anything with kernel updates.  It simply goes through /usr and /var and verify all of the files against the manifest of the version of Clear installed and make sure they are correct.  If they are not, swupd will repair them.

**Comment by mns6070** [comment]
Okay.  Just making sure.

**Comment by mns6070** [comment]
Everything should be fixed now.

**Comment by mvincerx** [comment]
LGTM. Thanks, all.

**Pull request merged**